<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sellerwindow</class>
 <widget class="QWidget" name="sellerwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>sellerwindow</string>
  </property>
  <widget class="QLabel" name="background">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>821</width>
     <height>600</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">Qlabel {
    border: none;
    background: none; /* Initial transparent green glow */
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../resources/bg1.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="openExternalLinks">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="neon">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>0</y>
     <width>341</width>
     <height>571</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">Qlabel {
    border: none;
    background: none; /* Initial transparent green glow */
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../resources/neon.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="openExternalLinks">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="addProductWidget" native="true">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>-10</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <widget class="QLabel" name="price">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>120</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Light</family>
      <pointsize>23</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
    border: none;
    background: none;
    color: white;
    font-family: 'Yu Gothic UI Light', sans-serif;
}</string>
    </property>
    <property name="text">
     <string>Цена:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="subcategoryChoose">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>380</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Light</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">/* Стили для контейнера select */
.custom-select {
    position: relative;
    display: inline-block;
    width: 200px; /* Ширина контейнера */
}

/* Скрываем оригинальный select */
.custom-select select {
    display: none; 
}

/* Стили для отображаемого элемента */
.select-selected {
    background-color: #f1f1f1;
    padding: 10px;
    border: 1px solid #ccc;
    border-radius: 5px;
    cursor: pointer;
}

/* Стили для выпадающего списка */
.select-items {
    position: absolute;
    background-color: #f9f9f9;
    border: 1px solid #ccc;
    border-radius: 5px;
    z-index: 99;
    display: none; /* Скрыт по умолчанию */
    max-height: 150px; /* Максимальная высота */
    overflow-y: auto; /* Прокрутка при переполнении */
}

/* Стили для элементов выпадающего списка */
.select-items div {
    padding: 10px;
    cursor: pointer;
}

/* Эффект при наведении на элементы */
.select-items div:hover {
    background-color: #ddd;
}

/* Стрелка для выпадающего списка */
.select-arrow {
    position: absolute;
    right: 10px;
    top: 14px;
    border: solid transparent;
    border-width: 5px 5px 0 5px;
    border-top-color: #333; /* Цвет стрелки */
}</string>
    </property>
   </widget>
   <widget class="QComboBox" name="categoryChoose">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>380</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Light</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">/* Стили для контейнера select */
.custom-select {
    position: relative;
    display: inline-block;
    width: 200px; /* Ширина контейнера */
}

/* Скрываем оригинальный select */
.custom-select select {
    display: none; 
}

/* Стили для отображаемого элемента */
.select-selected {
    background-color: #f1f1f1;
    padding: 10px;
    border: 1px solid #ccc;
    border-radius: 5px;
    cursor: pointer;
}

/* Стили для выпадающего списка */
.select-items {
    position: absolute;
    background-color: #f9f9f9;
    border: 1px solid #ccc;
    border-radius: 5px;
    z-index: 99;
    display: none; /* Скрыт по умолчанию */
    max-height: 150px; /* Максимальная высота */
    overflow-y: auto; /* Прокрутка при переполнении */
}

/* Стили для элементов выпадающего списка */
.select-items div {
    padding: 10px;
    cursor: pointer;
}

/* Эффект при наведении на элементы */
.select-items div:hover {
    background-color: #ddd;
}

/* Стрелка для выпадающего списка */
.select-arrow {
    position: absolute;
    right: 10px;
    top: 14px;
    border: solid transparent;
    border-width: 5px 5px 0 5px;
    border-top-color: #333; /* Цвет стрелки */
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="subcategory">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>310</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Light</family>
      <pointsize>23</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
    border: none;
    background: none;
    color: white;
    font-family: 'Yu Gothic UI Light', sans-serif;
}</string>
    </property>
    <property name="text">
     <string>Подкатегория</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>490</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgba(83, 208, 158, 0.2); 
    border: 2px solid #53D09E;
    border-radius: 20px;
	font-family: 'Yu Gothic UI Light', sans-serif;
    font-size: 20px;
text-align: center;
}
QPushButton:hover {
    color: rgba(83, 208, 158, 1)
}</string>
    </property>
    <property name="text">
     <string>Назад</string>
    </property>
   </widget>
   <widget class="QLabel" name="description">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>170</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Light</family>
      <pointsize>23</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
    border: none;
    background: none;
    color: white;
    font-family: 'Yu Gothic UI Light', sans-serif;
}</string>
    </property>
    <property name="text">
     <string>Описание:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="applyButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>490</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgba(83, 208, 158, 0.2); 
    border: 2px solid #53D09E;
    border-radius: 20px;
	font-family: 'Yu Gothic UI Light', sans-serif;
    font-size: 20px;
text-align: center;
}
QPushButton:hover {
    color: rgba(83, 208, 158, 1)
}</string>
    </property>
    <property name="text">
     <string>Подтвердить</string>
    </property>
   </widget>
   <widget class="QLabel" name="category">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>310</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Light</family>
      <pointsize>23</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
    border: none;
    background: none;
    color: white;
    font-family: 'Yu Gothic UI Light', sans-serif;
}</string>
    </property>
    <property name="text">
     <string>Категория</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="priceField">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>135</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic Light</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background-color: rgba(83, 208, 158, 0.2); 
    border: 2px solid #53D09E;
    border-radius: 20px;
    padding: 5px; /* Добавление отступа для текста */
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameField">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>80</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic Light</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background-color: rgba(83, 208, 158, 0.2); 
    border: 2px solid #53D09E;
    border-radius: 20px;
    padding: 5px; /* Добавление отступа для текста */
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="name">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>70</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Light</family>
      <pointsize>23</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
    border: none;
    background: none;
    color: white;
    font-family: 'Yu Gothic UI Light', sans-serif;
}</string>
    </property>
    <property name="text">
     <string>Название:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="descriptionField">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>190</y>
      <width>291</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic Light</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit {
    background-color: rgba(83, 208, 158, 0.2); 
    border: 2px solid #53D09E;
    border-radius: 20px;
    padding: 5px; /* Добавление отступа для текста */
    vertical-align: top; /* Выровнять текст по верхнему краю */
}</string>
    </property>
   </widget>
   <zorder>description</zorder>
   <zorder>subcategoryChoose</zorder>
   <zorder>price</zorder>
   <zorder>categoryChoose</zorder>
   <zorder>subcategory</zorder>
   <zorder>name</zorder>
   <zorder>category</zorder>
   <zorder>nameField</zorder>
   <zorder>descriptionField</zorder>
   <zorder>priceField</zorder>
   <zorder>applyButton</zorder>
   <zorder>cancelButton</zorder>
  </widget>
  <widget class="QLabel" name="introText">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>271</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI Light</family>
     <pointsize>33</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
    border: none;
    background: none;
    color: rgb(100, 249, 189);
    font-family: 'Yu Gothic UI Light', sans-serif;
}</string>
   </property>
   <property name="text">
    <string>Продавец</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>181</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI Light</family>
     <pointsize>27</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">/* Custom button style */
QPushButton {
    border: none;
    background: none;
    color: white;
    font-family: 'Yu Gothic UI Light', sans-serif;
}

QPushButton:hover {
    color: #53D09E
}</string>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>181</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI Light</family>
     <pointsize>27</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">/* Custom button style */
QPushButton {
    border: none;
    background: none;
    color: white;
    font-family: 'Yu Gothic UI Light', sans-serif;
}

QPushButton:hover {
    color: #53D09E
}</string>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="myProductsButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>191</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI Light</family>
     <pointsize>27</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">/* Custom button style */
QPushButton {
    border: none;
    background: none;
    color: white;
    font-family: 'Yu Gothic UI Light', sans-serif;
}

QPushButton:hover {
    color: #53D09E
}</string>
   </property>
   <property name="text">
    <string>Мои товары</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>420</y>
     <width>191</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI Light</family>
     <pointsize>27</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">/* Custom button style */
QPushButton {
    border: none;
    background: none;
    color: white;
    font-family: 'Yu Gothic UI Light', sans-serif;
}

QPushButton:hover {
    color: #53D09E
}</string>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>50</y>
     <width>461</width>
     <height>491</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
    background-color: rgba(45, 45, 45,0.4); 
    border-radius: 20px;
    padding: 5px; /* Добавление отступа для текста */
}
</string>
   </property>
  </widget>
  <widget class="QLabel" name="messageText">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>547</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI Light</family>
     <pointsize>20</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
    border: none;
    background: none;
    color: rgb(100, 249, 189);
    font-family: 'Yu Gothic UI Light', sans-serif;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextEdit" name="productInfo">
   <property name="geometry">
    <rect>
     <x>325</x>
     <y>65</y>
     <width>411</width>
     <height>451</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
    background: transparent; /* Прозрачный фон */
    color: white; /* Цвет текста, измените по необходимости */
    font-family: 'Yu Gothic UI Light', sans-serif;
}</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Yu Gothic UI Light','sans-serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Segoe UI'; font-size:16pt;&quot;&gt;Наименование продукта:&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Segoe UI'; font-size:16pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Segoe UI'; font-size:16pt;&quot;&gt;Цена:&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Segoe UI'; font-size:16pt;&quot;&gt;Описание:&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Segoe UI'; font-size:16pt;&quot;&gt;Артикул:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="right">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>460</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">Qlabel {
    border: none;
    background: none;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../resources/right-arrow.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="openExternalLinks">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="rightButton">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>460</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	 border: none;
    background: none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="left">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>460</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">Qlabel {
    border: none;
    background: none;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../resources/left-arrow.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="openExternalLinks">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="leftButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>460</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	 border: none;
    background: none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteProductButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>460</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: rgba(83, 208, 158, 0.2); 
    border: 2px solid #53D09E;
    border-radius: 20px;
	font-family: 'Yu Gothic UI Light', sans-serif;
    font-size: 20px;
text-align: center;
}
QPushButton:hover {
    color: rgba(83, 208, 158, 1)
}</string>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>textEdit</zorder>
  <zorder>neon</zorder>
  <zorder>addProductWidget</zorder>
  <zorder>introText</zorder>
  <zorder>addButton</zorder>
  <zorder>deleteButton</zorder>
  <zorder>myProductsButton</zorder>
  <zorder>backButton</zorder>
  <zorder>messageText</zorder>
  <zorder>productInfo</zorder>
  <zorder>right</zorder>
  <zorder>rightButton</zorder>
  <zorder>left</zorder>
  <zorder>leftButton</zorder>
  <zorder>deleteProductButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
